.github/workflows/scripts/fetch_packages_metadata.py

```diff
@@ -0,0 +1,75 @@
+from requests import get
```

Comment:
Due to it's location this Python file wasn't processed by the linter/black. Can you update https://github.com/elyra-ai/elyra/blob/main/Makefile#L111 to also process files in .github/workflows/scripts? This will surface a few import order issues in your code.

Reply:
added .github in this line